packages/amesos2/src/Amesos2_Superlu_FunctionMap.hpp

```diff
@@ -77,27 +77,67 @@ namespace SLU {
     typedef int int_t;
 #include "supermatrix.h"
 #include "slu_util.h"
+#include "superlu_enum_consts.h"
+
 
     namespace S {               // single-precision real definitions
+
+#ifdef HAVE_SUPERLU_GLOBALLU_T_ARG
+		typedef struct {
```

Comment:
@srajama1 The `GlobalLU_t` types are not available in the included Superlu files at lines 78-80, but in the respective `slu_*defs.h` files and are dependent on the data type (float, double, complex, complexdouble). The respective files are commented out at the end of each namespace and extern declarations are used; to preserve this I added the `GlobalLU_t` `typedef`s here. We can do this differently if there is a better way.